addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -301,6 +301,7 @@ export class WebsitePreview extends Component {
         this.iframe.el.contentDocument.addEventListener('keyup', ev => {
             this.iframe.el.dispatchEvent(new KeyboardEvent('keyup', ev));
         });
+        this.iframe.el.contentDocument.querySelector('#wrapwrap').classList.toggle('o_is_mobile', this.websiteService.context.isMobile);
```

Comment:
I think adding the `o_is_mobile` class on the iframe's #wrapwrap (once loaded) can be centralised in a useEffect on the client action:
```js
useEffect(() => {
    if (this.iframe.el.contentDocument.querySelector('#wrapwrap')) {
        this.iframe.el.contentDocument.querySelector('#wrapwrap').classList.toggle('o_is_mobile', this.websiteContext.isMobile);
    }
}, () => [this.websiteContext.isMobile]);
```
Therefore it would not be needed from the wysiwyg_adapter and the mobile preview systray item